src/map.rs

```diff
@@ -323,7 +365,7 @@ pub fn process_loaded_tile_maps(
         mut created_entities,
     ) in query.iter_mut()
     {
-        if new_meshes.contains_key(map_handle) {
+        if changed_maps.contains(map_handle) {
```

Comment:
Can no longer treat a new mesh as equivalent to an updated map, since individual tile images could change without needing a mesh to be rebuilt.